include/nds/dma.h

```diff
@@ -66,18 +66,20 @@ extern "C" {
 #define DMA_BUSY            BIT(31)
 #define DMA_IRQ_REQ         BIT(30)
 
-#define DMA_START_NOW       0
-#define DMA_START_CARD      (5 << 27)
+#define DMA_START_NOW       0         ///< Start DMA immediately
 
 #ifdef ARM7
-#    define DMA_START_VBL   BIT(27)
+#    define DMA_START_VBL   (1 << 28) ///< Auto-start DMA on vertical blank
```

Comment:
`1 << 27`?

Reply:
Actually, no! Both melonDS and gbatek point out that ARM7 only uses bits 28-29 for its DMA targets.

Reply:
(Though perhaps it may be a good idea to write a test for this, as such.)